Assets/MixedRealityToolkit.Examples/Demos/StandardShader/Scenes/ClippingExamples.unity

```diff
@@ -112,6 +112,245 @@ NavMeshSettings:
     debug:
       m_Flags: 0
   m_NavMeshData: {fileID: 0}
+--- !u!1 &50069567
```

Comment:
Instead of putting an experimental feature in an example, could you instead create a new scene under examples/experimental/pinchboxslider/Clippingbrain.unity, or something like this? Otherwise people may not realize that the pinch slider box is still experimental.

Reply:
I believe you used a similar pattern for the hand menu